corehq/apps/linked_domain/const.py

```diff
@@ -15,21 +15,29 @@
 MODEL_HMAC_CALLOUT_SETTINGS = 'hmac_callout_settings'
 MODEL_KEYWORD = 'keyword'
 
-LINKED_MODELS = [
+INDIVIDUAL_DATA_MODELS = [
     (MODEL_APP, ugettext_lazy('Application')),
+    (MODEL_FIXTURE, ugettext_lazy('Lookup Table')),
+    (MODEL_REPORT, ugettext_lazy('Report')),
+    (MODEL_KEYWORD, ugettext_lazy('Keyword')),
+]
+
+DOMAIN_LEVEL_DATA_MODELS = [
     (MODEL_USER_DATA, ugettext_lazy('Custom User Data Fields')),
     (MODEL_PRODUCT_DATA, ugettext_lazy('Custom Product Data Fields')),
     (MODEL_LOCATION_DATA, ugettext_lazy('Custom Location Data Fields')),
     (MODEL_ROLES, ugettext_lazy('User Roles')),
     (MODEL_FLAGS, ugettext_lazy('Feature Flags and Previews')),
-    (MODEL_FIXTURE, ugettext_lazy('Lookup Table')),
+]
+
+FEATURE_FLAG_DATA_MODELS = [
```

Comment:
I think it'll be useful to store the flags themselves here, whether as part of these tuples or as an additional dict.

Reply:
That code wasn't on my radar thanks. I like the idea of adding it as an additional dict in const.py.